### PR TITLE
Add Python 3 support

### PR DESCRIPTION
**Pull request by dan-blanchard**
This also updates the piwik search engine list.  Although, it should be noted that they do not make the same file we used to read available anymore, so I've had to hardcode the URL to be for version 2.14.3.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,4 +1,4 @@
 recursive-exclude tests *
 include *.txt
 include *.rst
-include serpextract/search_engines.pickle
\ No newline at end of file
+include serpextract/search_engines.pickle*
```
Comment (msukmanowsky) [inline MANIFEST.in]: Was the wildcard needed here?

Comment (msukmanowsky) [inline MANIFEST.in]: Ahh I see python 2/3. Mind naming them `search_engines.py2.pickle` and `search_engines.py3.pickle`?

**Comment by msukmanowsky** [comment]
:+1: looks good other than the naming on those pickle files.

File: MANIFEST.in (hunk 1 above)
Comment (dan-blanchard) [inline MANIFEST.in]: Sure thing. I'll fix that.

**Pull request merged**